src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -3790,7 +3790,10 @@ public function set_modifiable_text( string $plaintext_content ): bool {
 				 * properly escape these things, but this could mask regex patterns
```

Comment:
Does this comment need to be updated to reflect the double-escaped state protection?